packages/base-driver/lib/basedriver/desired-caps.js

```diff
@@ -74,7 +74,8 @@ validator.validators.isArray = function isArray(value) {
   return 'must be of type array';
 };
 validator.validators.deprecated = function deprecated(value, options, key) {
-  if (options) {
+  // do not print caps that hasn't been provided.
+  if (typeof value !== 'undefined' && options) {
```

Comment:
you could also use !_.isUndefined

Reply:
yea, i thought to use lodash but the file used this typeof method in the file wide, so this pr followed the method